uber/templates/registration/merch.html

```diff
@@ -22,11 +29,12 @@
             }
         }, 'json');
     }
+    l;
```

Comment:
This is perhaps not supposed to be an `l`?

Reply:
whoops, good catch, I must have fat-fingered that!